### PR TITLE
add interop start timestamp explainer to spec docs

### PR DESCRIPTION
**Pull request by AmadiMichael**
**Description**

This pr adds the concept of Interop Start Timestamp to the Specs.

**Metadata**

- Fixes a part of [this issue](https://github.com/ethereum-optimism/optimism/issues/11367)

### CODE REVIEW

File: specs/interop/predeploys.md [hunk 1]
```diff
@@ -117,6 +129,7 @@ A simple implementation of the `executeMessage` function is included below.
 function executeMessage(Identifier calldata _id, address _target, bytes calldata _msg) public payable {
     require(_id.timestamp <= block.timestamp);
     require(L1Block.isInDependencySet(_id.chainid));
+    require(_id.timestamp > L1Block.interopStart());
```
Comment (tynes) [inline specs/interop/predeploys.md]: We could choose to put this value directly in the CrossL2Inbox to save a call, what do you think?

**Comment by tynes** [comment]
This doesn't match the implemention exactly in the pseudocode

https://github.com/ethereum-optimism/optimism/pull/11398

We should migrate more pseudocode to natural language when it makes sense to but no need to do that now for this

I think the implementation it the right way to do it

File: specs/interop/predeploys.md (hunk 1 above)
Comment (AmadiMichael) [reply specs/interop/predeploys.md]: Yeah, I think its worth the gas savings

Comment (AmadiMichael) [reply specs/interop/predeploys.md]: Interestingly, this already exists in Crossl2Inbox and not L1Block, it's my mistake in the reference implementation

**Review by tynes — APPROVED** [review]

**Pull request merged**